CMakeLists.txt

```diff
@@ -108,6 +112,21 @@ find_package(UV REQUIRED)
 
 include(cmake/flags.cmake)
 
+if (WITH_TLS)
+    find_package(OpenSSL REQUIRED)
+
+    add_definitions(/DCPPHTTPLIB_OPENSSL_SUPPORT)
+
+    if (OPENSSL_FOUND)
+        include_directories(${OPENSSL_INCLUDE_DIR})
```

Comment:
`include_directories(${OPENSSL_INCLUDE_DIR})` shouldn't be necessary when using `OpenSSL::SSL`and `OpenSSL::Crypto` in `target_link_libraries`

Reply:
Seems that for MVSC build this is still needed, can't find a better solution.